res/app/control-panes/device-control/device-control.jade

```diff
@@ -25,7 +25,7 @@
 
           ul.dropdown-menu(role='menu', data-toggle='dropdown', ng-show='groupDevices.length > 0').pointer.unselectable
             li(ng-repeat='groupDevice in groupDevices')
-              a(ng-click='controlDevice(groupDevice); $event.stopPropagation()')
+              a.stf-device-control(ng-click='controlDevice(groupDevice); $event.stopPropagation()')
```

Comment:
@gunta would know better I suppose, but is there a reason why you reused the `stf-device-control` class here? It seems to be reserved for the top level container in this template. If there's no particular reason, could you rename the classname and then we can finally merge this?